comp110_architecture.tex

```diff
@@ -5,41 +5,150 @@
 
 \section{Introduction}
 
-Write your introduction here. A brief introduction is recommended, which should outline key details of the chosen topic and the reviewed papers, motivate the work, and provide a roadmap of key points to the reader. The motivation is quite important here, as essays should have a contribution (i.e., what is the point of the essay, and what does the reader take away from the essay) and the link between the motivation (in the introduction) and the contribution (in the conclusion) should be made clear.
-
-\section{Your section title here}
-
-Write the main body of your essay here. Add more sections if appropriate. You may choose to write about each of your three papers in its own section, or you may choose a different structure. Either way, remember that you are being assessed on technical insight and analysis: it is not enough to merely summarise the contents of the three papers. You must demonstrate the ability to make inferences beyond what is written in the papers, and to draw the three papers together into a single coherent narrative.
+Procedural content generation (PCG) is one of many methods used today to create and design levels, particularity 2D platformers. PCG not only creates a new experience for the player with each playthrough, but also when done correctly can keep development costs down, by removing the need for extra level designers\cite{walaa}. The only real issue with PCG is which one to choose, with the variety of algorithms that are available it can be daunting to decide on which direction to go. So with PCG its important to have a clear idea of the task you want to accomplish as this will clearly define which algorithm would be the best for that game. With this in mind this paper contains three different algorithms that could be used in a 2D Platform game. 
+
+\subsection{Motivation}
+The three examples used are to be discussed on which algorithm would be the easiest to implement while still being the most efficient. They are as follows:
+\newline 
+In section 2: An Iterative Content Adaptation Algorithm (ICA) used to enhance difficulty of a game\cite{fausto}.
+\newline
+Section 2.1: Genetic Algorithm (GA) used alongside an Agent-based Digger Algorithm to use and add assets to an established game\cite{walaa}.
+\newline
+Section 2.2: An Algorithm used alongside the Launchpad to create a whole new game\cite{web}.
+\newline
+In section 3: A comparison of efficiency will be drawn based on their respective results.
+\newline
+More detailed information can be found here \cite{fausto,walaa,web} respectively. Aa brief a description as possible for each algorithm and how they work are presented in the following sections. Looking at ease of implementation the obvious choice would be the ICA due to its simplicity and ease of use on more than one game, more on this in section 2.
+
+\section{Iterative Content Algorithm (ICA)}
```

Comment:
This subsection is very long. Avoid being too descriptive -- take advantage of psueudo-code and figures to reduce words used to merely describe an algorithm. We are looking for evidence of analysis and evaluation.